src/Sylius/Behat/Service/DriverHelper.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Service;
+
+use Behat\Mink\Driver\DriverInterface;
+use Behat\Mink\Driver\Selenium2Driver;
+use DMore\ChromeDriver\ChromeDriver;
+
+abstract class DriverHelper
```

Comment:
what about
```suggestion
final class DriverHelper
```
with a private constructor, so the class won't be initializable?

Reply:
I've used the same construction as in `JQueryHelper` but I agree, we could change them both 👍